main/sceneBuilder.h

```diff
@@ -0,0 +1,68 @@
+#ifndef SCENEBUILDER_H
+#define SCENEBUILDER_H
+
+#include "constants.h"
+#include "core/vec3.h"
+#include "core/ray.h"
+#include "obj/generators.h"
+#include "obj/objects.h"
+
+#include <memory>
+#include <string>
+#include <string_view>
+#include <vector>
+
+
+class SceneBuilder
```

Comment:
Please add a comment what this class is supposed to do, for example:

> Manages all objects in one scene, and allows to ...

(whatever that is).

Reply:
ok